views/abandoned-cart.ejs

```diff
@@ -61,38 +61,58 @@
                                       valign="top"
                                       bgcolor="">
                                       <div style="text-align: center;"><span style="color:#333333;">
-                                        <p><strong><%= customer.name.given_name %></strong>, <%= i18n.cart_text %></p>
+                                        <p><%= i18n.cart_text %></p>
                                         </span></div>
                                   </td>
                                 </tr>
                               </table>
 
+                              <table border="0" cellpadding="0" cellspacing="10" class="module" data-role="module-button" data-type="button" role="module" style="table-layout:fixed" width="100%"><tbody><tr><td align="center" class="outer-td" style="padding:10px 0px 15px 0px"><table border="0" cellpadding="0" cellspacing="0" class="button-css__deep-table___2OZyb wrapper-mobile" style="text-align:center"><tbody><tr><td align="center" bgcolor="<%= themeColor %>" class="inner-td" style="border-radius:6px;font-size:16px;text-align:center;background-color:inherit"><a class="btn" style="background-color:<%= themeColor %>;border:1px solid #333333;border-color:#FFFFFF;border-radius:3px;border-width:1px;color:#FFFFFF;display:inline-block;font-family:arial,helvetica,sans-serif;font-size:16px;font-weight:normal;letter-spacing:0px;line-height:16px;padding:12px 70px 12px 70px;text-align:center;text-decoration:none" href="<%= cart.permalink %>" target="_blank"><%= i18n.cart_cta %></a></td></tr>
+                              <tr>
+                                <td style="padding: 30px 45px 0px 45px;line-height:22px;text-align:inherit;"
+                                    height="100%"
+                                    valign="top"
+                                    bgcolor="">
+                                    <div style="text-align: center;"><span style="color:#333333;">
+                                      <p><%= i18n.or %> <a href="<%= store.homepage %>/app/#/account/orders"><%= i18n.visit_store %></a> </p>
```

Comment:
shoud use domain instead of homepage to concat with custom page url

```suggestion
                                      <p><%= i18n.or %> <a href="https://<%= store.domain %>/app/#/account/orders"><%= i18n.visit_store %></a> </p>
```

Reply:
Actually in this cases it should go to the home page, and not the user's. Changed it when going to "my account" page.